2019-10-03 15:44:55,467 INFO: Microblog startup [in /home/meth/dev-practice/py-practice/microblog/app/__init__.py:42]
2019-10-03 15:45:18,153 ERROR: Exception on /edit_profile [GET] [in /home/meth/dev-practice/py-practice/microblog/venv/lib/python3.5/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/meth/dev-practice/py-practice/microblog/venv/lib/python3.5/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/meth/dev-practice/py-practice/microblog/venv/lib/python3.5/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/meth/dev-practice/py-practice/microblog/venv/lib/python3.5/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/meth/dev-practice/py-practice/microblog/venv/lib/python3.5/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/meth/dev-practice/py-practice/microblog/venv/lib/python3.5/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/meth/dev-practice/py-practice/microblog/venv/lib/python3.5/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/meth/dev-practice/py-practice/microblog/venv/lib/python3.5/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/meth/dev-practice/py-practice/microblog/app/routes.py", line 80, in edit_profile
    form = EditProfileForm(current_user.username)
  File "/home/meth/dev-practice/py-practice/microblog/venv/lib/python3.5/site-packages/wtforms/form.py", line 212, in __call__
    return type.__call__(cls, *args, **kwargs)
  File "/home/meth/dev-practice/py-practice/microblog/app/forms.py", line 36, in __init__
    self.original_username = original_username
NameError: name 'original_username' is not defined
2019-10-03 15:46:46,269 INFO: Microblog startup [in /home/meth/dev-practice/py-practice/microblog/app/__init__.py:42]
2019-10-03 16:05:07,762 INFO: Microblog startup [in /home/meth/dev-practice/py-practice/microblog/app/__init__.py:42]
2019-10-03 16:05:37,243 INFO: Microblog startup [in /home/meth/dev-practice/py-practice/microblog/app/__init__.py:42]
2019-10-03 16:05:50,344 INFO: Microblog startup [in /home/meth/dev-practice/py-practice/microblog/app/__init__.py:42]
2019-10-03 16:39:04,965 INFO: Microblog startup [in /home/meth/dev-practice/py-practice/microblog/app/__init__.py:42]
2019-10-03 16:39:26,840 INFO: Microblog startup [in /home/meth/dev-practice/py-practice/microblog/app/__init__.py:42]
2019-10-03 16:41:00,157 INFO: Microblog startup [in /home/meth/dev-practice/py-practice/microblog/app/__init__.py:42]
2019-10-03 16:41:20,681 INFO: Microblog startup [in /home/meth/dev-practice/py-practice/microblog/app/__init__.py:42]
2019-10-03 16:42:11,368 INFO: Microblog startup [in /home/meth/dev-practice/py-practice/microblog/app/__init__.py:42]
2019-10-03 16:43:06,078 INFO: Microblog startup [in /home/meth/dev-practice/py-practice/microblog/app/__init__.py:42]
2019-10-03 16:44:45,994 INFO: Microblog startup [in /home/meth/dev-practice/py-practice/microblog/app/__init__.py:42]
2019-10-03 17:19:31,785 INFO: Microblog startup [in /home/meth/dev-practice/py-practice/microblog/app/__init__.py:42]
2019-10-03 17:40:00,697 INFO: Microblog startup [in /home/meth/dev-practice/py-practice/microblog/app/__init__.py:42]
